requirements.txt

```diff
@@ -27,6 +27,7 @@ pycodestyle==2.4.0
 pyflakes==2.0.0
 PyJWT==1.7.0
 pylint==2.2.2
+python-social-auth==0.3.6
```

Comment:
Do we require this? or are we using `social-auth-app-django==3.1.0`?

Reply:
Actually, it is not necessary. I have removed it.